### PR TITLE
BUGFIX: TMA-358 Support replacing date dimension in execution contexts

### PR DESCRIPTION
**Pull request by vhtien**

### CODE REVIEW

**Comment by panjan** [comment]
Can we have a test for this?

**Comment by vhtien** [comment]
we do have a test for it. But the impact area of this function is very large. We cannot catch all things

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-rubocop-zuul/477/ : SUCCESS in 16s
- https://ci.intgdc.com/job/gooddata-ruby-tests-zuul/415/ : FAILURE in 42m 52s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/151/ : FAILURE in 8m 51s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-rubocop-zuul/479/ : SUCCESS in 10s
- https://ci.intgdc.com/job/gooddata-ruby-tests-zuul/417/ : SUCCESS in 42m 44s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/153/ : FAILURE in 4s

**Comment by vhtien** [comment]
ok to test

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-rubocop-zuul/480/ : SUCCESS in 13s
- https://ci.intgdc.com/job/gooddata-ruby-tests-zuul/418/ : SUCCESS in 43m 16s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/154/ : FAILURE in 13s

File: lib/gooddata/models/project.rb [hunk 1]
```diff
@@ -1328,6 +1328,23 @@ def replace_from_mapping(mapping, opts = {})
         end
       end
 
+      GoodData.logger.info 'Replacing dashboard saved views'
```
Comment (panjan) [inline lib/gooddata/models/project.rb]: Can we extract this into a method?

Comment (panjan) [inline lib/gooddata/models/project.rb]: Also, can we write a test for it?

**Pull request merged**